build.sbt

```diff
@@ -1,12 +1,18 @@
 import Dependencies._
 import sbtcrossproject.CrossPlugin.autoImport.crossProject
+def localSnapshotVersion = "2.2.3-SNAPSHOT"
+def isCI = System.getenv("CI") != null
```

Comment:
It will be true in TravisCI https://docs.travis-ci.com/user/environment-variables/#default-environment-variables